diff --git a/test/performance/hilbert.cu b/test/performance/hilbert.cu
index c43d8d1..c385ce9 100644
--- a/test/performance/hilbert.cu
+++ b/test/performance/hilbert.cu
@@ -64,7 +64,7 @@ decodeSfcKeys(cudaStream_t stream, const KeyType* keys, unsigned* x, unsigned* y
 int main()
 {
     using IntegerType = uint64_t;
-    unsigned numKeys  = 32000000;
+    unsigned numKeys  = 32000;
 
     using Real = double;
     Box<Real> box(-1, 1);
@@ -107,8 +107,9 @@ int main()
         { computeSfcKeys(stream, rawPtr(mortonKeys), rawPtr(dx), rawPtr(dy), rawPtr(dz), numKeys); };
 
         float t_hilbert = timeGpu(computeHilbert);
-        float t_morton  = timeGpu(computeMorton);
         std::cout << "compute time for " << numKeys << " hilbert keys: " << t_hilbert / 1000 << " s" << std::endl;
+        return 0;
+        float t_morton  = timeGpu(computeMorton);
         std::cout << "compute time for " << numKeys << " morton keys: " << t_morton / 1000 << " s" << std::endl;
 
         thrust::device_vector<unsigned> dx2(numKeys);
